CHANGELOG

```diff
@@ -1,4 +1,14 @@
 # Changelog
+## [0.2.2] - 2019-02-03
```

Comment:
It is 2020 already and it is also not 3 February anymore. 0.2.1 also used the wrong year, please update it as well. :-)

Reply:
Ooops, sorry. It seems my mind is not in 2020 yet :-)

Reply:
Done, thanks!